packages/iframe-stamper/README.md

```diff
@@ -55,3 +55,31 @@ const publicKey = await iframeStamper.init();
 // Injects a new private key in the iframe
 const injected = await iframeStamper.injectKeyExportBundle(exportBundle);
 ```
+
+Auth
+
```

Comment:
I think this can be deleted/consolidated with the first section. The code snippet is exactly the same :)